R/select_best.R

```diff
@@ -32,7 +32,8 @@
 #' least one term is required for these two functions. See the examples below.
 #' @param eval_time A numeric vector of time points where dynamic event time
```

Comment:
This is where I hate word wrapping. 

We should expect a single time point (or null) for these functions as well as `fit_best()`. (right?)

So how about: 

> A single numeric time point where dynamic event time metrics should be assessed (e.g., the time-dependent ROC curve, etc). The value should be consistent with the values used to create `x`. The `NULL` default will automatically use the first evaluation time used by `x`.